pkg/server/auth/server.go

```diff
@@ -490,3 +490,7 @@ func contains(ss []string, s string) bool {
 
 	return false
 }
+
+func Error(rw http.ResponseWriter, err string, code int) {
```

Comment:
Rather than making the string like this, you could marshal an anonymous struct...

```Go
    response := struct {
        Message string `json:"message"`
    }{Message: "testing"}
    b, err := json.Marshal(response)
```

Also, you should probably set the content-type

Reply:
You aren't controlling the message https://github.com/weaveworks/weave-gitops/pull/1844/files#diff-f96b9f66364af6512f2689b1500deebf882f76fc92556b66bc74f3c40005940aR250 which means that you could get something that didn't simply marshal like this.